src/pages/Rockets.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchAllRockets, toggleReservation } from '../redux/rockets/rocket';
+import './style.css';
+
+function Rockets() {
+  const rockets = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+
+  const reserveRocket = (id) => {
+    dispatch(toggleReservation(id));
+  };
+
+  useEffect(() => {
+    dispatch(fetchAllRockets());
+  }, []);
+  return (
+    <div className="rocket-container">
+      {rockets.map((item) => (
+        <div className="rocket-card" key={item.id}>
+          <div className="rocket-profile">
+            <img src={item.flickr_images} alt="" />
```

Comment:
- [ ] I think it's better to use the same width for all your images. Will create a better UI.

<img width="364" alt="Screen Shot 2022-11-19 at 12 58 33 PM" src="https://user-images.githubusercontent.com/47673311/202864357-692823df-6b64-4ea1-9cbc-bbb0e5b8adf3.png">

Reply:
- The project doesn't require us to have responsiveness